Highlight support for all HTTP methods in proxies (#391)
toddledev/documentation: docs/11-pages/06-proxies/index.md

# Configure proxies

> You can use proxies to map one URL to another URL. Using a proxy hides the path of the destination on the server, so that it appears that the location does not change. A proxy supports all HTTP methods (GET, POST, PUT, DELETE, etc.) and uses the same method, headers and body as the original request when forwarding the request to the destination.

Reviewer: ```suggestion
You can use proxies to map one URL to another URL. Using a proxy hides the path of the destination on the server, so that it appears that the location does not change. A proxy in Nordcraft supports all HTTP methods (GET, POST, PUT, DELETE, etc.) and uses the same method, headers and body as the original request when forwarding the request to the destination.
```

> You may want to use proxies to serve content from domains that are no longer in use, but where the content is still relevant. You may also want to use proxies to serve utility content like sitemap.xml files, robot.txt files, and ads.txt files. A proxy could also be used to send content to a different service, but on your own origin - for instance to a GraphQL service.

Reviewer: ```suggestion
You may want to use proxies to serve content from domains that are no longer in use, but where the content is still relevant. You may also want to use proxies to serve utility content like sitemap.xml files, robot.txt files, and ads.txt files. A proxy could also be used to send content to a different service, but on your own origin, for instance to a GraphQL service.
```

## Testing

> In order to verify that your proxy works as intended, click the "Preview page in new tab", or navigate to the URL that you have set up in the source section. If the proxy is configured correctly, you should see the desired location. If the proxy is not for a `GET` request, you might need to use a tool like Hoppscotch or Postman to test your new proxy.

Reviewer: ```suggestion
In order to verify that your proxy works as intended, click the "Preview page in new tab", or navigate to the URL that you have set up in the source section. If the proxy is configured correctly, you should see the desired location. 

:::tip
If the proxy uses and HTTP method that is not a `GET` request, you may need to use a tool like Hoppscotch or Postman to test your new proxy.
:::
```
